Do not enter src/quick for checks; we do not build this directory in this port
and it expects qt5-gui to be installed.
--- configure.json.orig	2018-06-15 11:30:15 UTC
+++ configure.json
@@ -1,6 +1,5 @@
 {
     "subconfigs": [
-        "src/qml",
-        "src/quick"
+        "src/qml"
     ]
 }
